deepchem/models/tests/test_dtnn.py

```diff
@@ -0,0 +1,38 @@
+import os
+import numpy as np
+import pytest
+from scipy import io as scipy_io
+
+from deepchem.data import NumpyDataset
+
+from deepchem.models.torch_models import DTNNModel
```

Comment:
Use try and except block to import tests.
This breaking tensorflow and jax test as you are trying to import torch in them

Reply:
Done